roles/openshift_node/tasks/upgrade/restart.yml

```diff
@@ -57,13 +57,45 @@
   retries: 30
   until: node_upgrade_oc_csr_approve is succeeded
 
-- name: Check status of node service
-  async_status:
-    jid: "{{ node_service.ansible_job_id }}"
```

Comment:
I don't think its a good idea to drop `async_status` - why would we want to re-implement it with `systemctl status atomic-openshift-node` and restart it if it has failed?

Reply:
Is it possible to use until: with async_status:?